### ISSUE TITLE
autoreconf fails under FreeBSD

### ISSUE THREAD

**Issue by graemeg** [manitou-mail/manitou-mail-ui#3]
I'm not sure if Manitou-Mail is 100% cross-platform. Being Qt based, I can't see why it shouldn't be. Anyway, I got the latest code from git today and tried to run the 'autoreconf' command as the README file mentioned.

This is the output I got:

```
[manitou-mail-ui (master)]$ autoreconf
aclocal-1.15: warning: autoconf input should be named 'configure.ac', not 'configure.in'
automake-1.15: warning: autoconf input should be named 'configure.ac', not 'configure.in'
configure.in:4: warning: AM_INIT_AUTOMAKE: two- and three-arguments forms are deprecated.  For more info, see:
configure.in:4: http://www.gnu.org/software/automake/manual/automake.html#Modernize-AM_005fINIT_005fAUTOMAKE-invocation
configure.in:7: error: required file './config.guess' not found
configure.in:7:   'automake --add-missing' can install 'config.guess'
configure.in:7: error: required file './config.sub' not found
configure.in:7:   'automake --add-missing' can install 'config.sub'
configure.in:4: error: required file './install-sh' not found
configure.in:4:   'automake --add-missing' can install 'install-sh'
configure.in:4: error: required file './missing' not found
configure.in:4:   'automake --add-missing' can install 'missing'
automake-1.15: warning: autoconf input should be named 'configure.ac', not 'configure.in'
src/Makefile.am:66: warning: '%'-style pattern rules are a GNU make extension
src/Makefile.am:69: warning: '%'-style pattern rules are a GNU make extension
src/Makefile.am: error: required file './depcomp' not found
src/Makefile.am:   'automake --add-missing' can install 'depcomp'
translations/Makefile.am:6: warning: '%'-style pattern rules are a GNU make extension
autoreconf-2.69: automake failed with exit status: 1
```

My system details are as follows:

```
[manitou-mail-ui (master)]$ uname -a
FreeBSD graeme-desktop 10.1-RELEASE FreeBSD 10.1-RELEASE #0 r274401: Tue Nov 11 21:02:49 UTC 2014     root@releng1.nyi.freebsd.org:/usr/obj/usr/src/sys/GENERIC  amd64
```

**Comment by manitou-mail** [comment]
This should be `autoreconf -i` to create the missing files.
Aside from that, the Qt binaries appear to be suffixed by _-qt4_ in the qt4 FreeBSD package, and the current configure script won't find them under these names. A fix will be committed shortly for that.

**Comment by manitou-mail** [comment]
This should be fixed by commit 89df476.
Also gmake should be used (as opposed to BSD make).
